priam/src/main/java/com/netflix/priam/config/IBackupRestoreConfig.java

```diff
@@ -97,4 +97,18 @@ default int getBackupVerificationSLOInHours() {
     default boolean enableV2Restore() {
         return false;
     }
+
+    /**
+     * Build the instance from backups by using restore process in case of an instance replacements.
+     * Note that we prefer this when data size is HUGE. C* streaming is super slow and for instances
+     * with big data size can lead to C* streaming for multiple days. Note that this is a little bit
+     * dangerous as you "will" some of the writes accepted by old instance but not uploaded to
+     * backup file system. Also we do not plan to run local repair on the replaced instance, so data
```

Comment:
I agree that not running repair is acceptable for a first iteration. Hypothetically though, how would we do it?

Reply:
Ideally, we should be deferring that task to the repair service. Where that repair service sits, how it gets executed is a different conversation though.